src/layouts/ContestLayout.js

```diff
@@ -154,8 +168,17 @@ const ContestLayout = (props) => {
 };
 export default ContestLayout;
 
-export const pageQuery = graphql`
-  query ContestsId($contestId: Int) {
+export const myQuery = graphql`
```

Comment:
Should we be naming these queries something more descriptive and connected to the component? Like maybe `contestLayoutQuery`? I'm not sure about good naming conventions for graphql.